src/main/src/main/java/org/geoserver/catalog/CascadedLayerInfo.java

```diff
@@ -0,0 +1,43 @@
+/* (c) 2019 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.catalog;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import org.geotools.styling.Style;
+
+/** @author ImranR */
```

Comment:
Useless java doc, we have git blame. Javadoc  describing what this interfaces does and on its methods needs to be added.

Reply:
will remove it